src/heroes/doomfist.opy

```diff
@@ -0,0 +1,138 @@
+#!include "ow2_constants.opy"
+
+
+rule "Doom shields":
+    @Event eachPlayer
+    @Hero doomfist
+    @Condition eventPlayer.getHealthOfType(Health.SHIELDS) == 100
+    
+    eventPlayer.setStatusEffect(eventPlayer, Status.UNKILLABLE, 9999)
+
+
+rule "Doom shields":
+    @Event eachPlayer
+    @Hero doomfist
+    @Condition eventPlayer.getHealthOfType(Health.SHIELDS) < 100
+    
+    eventPlayer.clearStatusEffect(Status.UNKILLABLE)
+
```

Comment:
What's the purpose of these rules?

Reply:
I just had them when I was testing, I forgot to remove them